UnityProject/Assets/Scripts/Util/SweetExtensions.cs

```diff
@@ -82,7 +82,11 @@ public static T GetRandom<T>(this List<T> list)
 
 	public static uint NetId(this GameObject go)
 	{
-		return go ? go.GetComponent<NetworkIdentity>().netId : global::NetId.Invalid; //maxValue is invalid (see NetId.cs)
+		return go == null
```

Comment:
I like ternaries but in this case the nested ternaries might be hard to read for people with less experience. I'd suggest using a simple if statement in this case.